tasks.py

```diff
@@ -341,6 +341,20 @@ def post_upgrade(context):
     run_command(context, command)
 
 
+@task()
+def dumpdata(context, format="json"):
+    """Dump data from database to db_output file."""
+    command = f"nautobot-server dumpdata -e extras.job --indent 4 -o db_output.{format} --format {format}"
```

Comment:
This may require you to wrap the command to two lines, but I'd rather spell out the args in full:

```suggestion
    command = f"nautobot-server dumpdata --exclude extras.job --indent 4 --output db_output.{format} --format {format}"
```